frontend/src/pages/auth/components/sign-up-form/styles.module.scss

```diff
@@ -0,0 +1,43 @@
+.auth-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 280px;
+}
+
+.auth-form .greeting {
```

Comment:
use more common names, like title instead of greeting

Reply:
done